source/_posts/binarized-neural-network.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Link to previous content - in content there should be link with previous content